docs/sla.rst

```diff
@@ -1,30 +0,0 @@
-=======================
```

Comment:
I would keep this. Let's add an outdated note for now and update it with the current SLA.